corehq/apps/reports/templates/reports/standard/base_template.html

```diff
@@ -30,7 +30,6 @@
     <!-- Scripts that are specific to a single report or interface -->
     <script src="{% static 'data_interfaces/js/case_management.js' %}"></script>
     <script src="{% static 'data_interfaces/js/archive_forms.js' %}"></script>
-    <script src="{% static 'hqpillow_retry/js/pillow_errors.js' %}"></script>
```

Comment:
A random staff private report that includes javscript on every report? classic reporting code

Reply:
The reports UI has rather a lot of baggage at this point.